ONSdigital/respondent-home-ui
# Default server task to use production config and turn off DEBUG

# Motivation and Context

A few quick fixes to turn off debug when using ProductionConfig.

# How to test?



`make serve`  should spin up four gunicorn workers 

NB: will require .env (try: [local.env](https://github.com/ONSdigital/respondent-home-ui/blob/521031cc292df32681dcb2282ae703dd916de26f/tests/test_data/local.env) from the integration branch).
